easybuild/easyconfigs/s/Simpack/Simpack-2017.eb

```diff
@@ -0,0 +1,20 @@
+# Author: Alexi Rivera
+# C3SE / Chalmers University of Technology
+
+name = 'Simpack'
+version = '2017'
+
+homepage = 'www.simpack.com'
+
+description = """ SIMPACK is a general purpose Multi-Body Simulation (MBS) software which
+ is used for the dynamic analysis of any mechanical or mechatronic system. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCE_TAR_GZ]
```

Comment:
please provide some info on where to download (probably requires registration)?

Reply:
Not sure what information to provice, since it is a commercial software that needs to be bought and cannot be downloaded from any page as such.

Reply:
Just include a comment stating this: `# commercial software, requires license to download, see http://www.simpack.com`.